README.md

```diff
@@ -53,6 +53,7 @@ At most 5 kube-state-metrics releases will be recorded below.
 | **v1.0.x** |  4.0.0-beta.0     |          ✓          |         ✓          |        ✓           |         ✓          |         -          |         -          |
 | **v1.1.0** |  release-5.0      |          ✓          |         ✓          |        ✓           |         ✓          |         ✓          |         -          |
 | **v1.2.0** |  v6.0.0           |          ✓          |         ✓          |        ✓           |         ✓          |         ✓          |         ✓          |
+| **v1.3.0** |  v6.0.0           |          ✓          |         ✓          |        ✓           |         ✓          |         ✓          |         ✓          |
```

Comment:
* Should we deprecate v0.4.0 version for kube-state-metrics as we have stated that `At most 5 kube-state-metrics releases will be recorded below.`
* Just in wonder that should we deprecate the supporting info for Kubernetes 1.4 and Kubernetes 1.5 and add the support info for Kubernetes 1.10 since it has been released.

@brancz